pkg/virtualization/core/storage/service/ImageManagementService_test.go

```diff
@@ -34,7 +35,7 @@ func TestCreateDynamicVirtualHardDisk(t *testing.T) {
 		t.Fatal("Failed " + err.Error())
 	}
 	path := "c:\\test\\tmp.vhdx"
-	setting, err := disk.GetVirtualHardDiskSettingData(whost, path, 512, 512, 0, 1024*1024*10, true)
+	setting, err := disk.GetVirtualHardDiskSettingData(whost, path, 512, 512, 0, 1024*1024*10, true, disk.VirtualHardDiskFormat_2)
```

Comment:
Out of curiosity, which format is disk.VirtualHardDiskFormat_2 referring to ? I just need some context on the mapping for personal knowledge.

Reply:
This actually refers to the internal code for VHDX which is 2. I did not change this name since this was added by someone else and most probably will have implications somewhere else in code.